celestia-node.md

```diff
@@ -23,40 +23,40 @@ make install
 ```
 
 
-## Full Node Configuration
+## Bridge Node Configuration
 
 ### Getting trusted hash
 > Caveat: You need a running celestia-app in order to continue this guideline. Please refer to [celestia-app.md](https://github.com/celestiaorg/networks/celestia-app.md) for installation.
 
 
-You need to have the trusted hash in order to initialize the Celestia full node
+You need to have the trusted hash in order to initialize the Celestia bridge node
 In order to know the hash, you need to query the celestia-app:
 ```sh
 curl -s http://localhost:26657/block?height=1 | grep -A1 block_id | grep hash
 ```
 
-### Running full node
-#### 1. Initialize the full node
+### Running bridge node
+#### 1. Initialize the bridge node
 ```sh
-celestia full init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
+celestia bridge init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
 ```
 
 Example:
 ```sh 
-celestia full init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 4632277C441CA6155C4374AC56048CF4CFE3CBB2476E07A548644435980D5E17
+celestia bridge init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 4632277C441CA6155C4374AC56048CF4CFE3CBB2476E07A548644435980D5E17
```

Comment:
Fix: remove `--headers.trusted-hash` as celestia bridge cli doesn't have this flag. 
I got a `Error: unknown flag: --headers.trusted-hash` flag while running this

Reply:
This is as a result of an existing bug on the `celestia-cli` that will be fixed as part of the [PR mentioned above](https://github.com/celestiaorg/celestia-node/issues/389).